common/slot_clock/src/lib.rs

```diff
@@ -105,11 +105,11 @@ pub trait SlotClock: Send + Sync + Sized + Clone {
     }
 
     /// Returns the `Duration` since the start of the current `Slot`. Useful in determining whether to apply proposer boosts.
-    fn seconds_from_current_slot_start(&self, seconds_per_slot: u64) -> Option<Duration> {
+    fn duration_from_current_slot_start(&self) -> Option<Duration> {
```

Comment:
Since the precision of the duration is limited to seconds, this should be kept as part of the method name

Reply:
change reverted